gendarme-ignore.txt

```diff
@@ -16,6 +23,10 @@ R: Gendarme.Rules.BadPractice.PreferSafeHandleRule
 T: Java.Interop.JniEnvironmentInfo
 T: Java.Interop.JniRuntime
 
+R: Gendarme.Rules.Correctness.AvoidFloatingPointEqualityRule
+# This is in from generated code. We could plausibly fix the generated code, but I'm not at all sure what to change it *to*.
+M: System.Int32 Java.Interop.JavaSingleArray::IndexOf(System.Single)
+M: System.Int32 Java.Interop.JavaDoubleArray::IndexOf(System.Double)
 
```

Comment:
I mean the usage of the method itself. Whether it is going to be used with given values or calculated values. I guess it is not defined?